docs/_data/prague-2021-schedule.yaml

```diff
@@ -0,0 +1,79 @@
+writing_day:
+- time: '10:00'
+  title: 'Platform opens'
+- time: '11:00'
+  title: Morning introduction - Introduction to Sprints & Project Introductions
+- time: '11:15'
+  title: Team up and start working
+- time: '14:30'
+  title: Lunch Break
+- time: '19:00'
+  title: End of Writing Day
+
+talks_day1:
+- time: '9:00'
+  title: Platform opens
+- time: '10:00'
+  title: Write the Docs Team - Introduction to Write the Docs
+- time: '10:45'
+  title: <b>Unconference Starts</b>
+- time: '10:45'
+  slug: hitchhiker-s-guide-to-documentation-tools-and-processes-lukas-reu-ner
+- time: '11:25'
+  title: Break (10 mins)
+- time: '11:35'
+  slug: more-than-words-reviewing-and-updating-your-information-architecture-lana-brindley-she-her
+- time: '12:20'
+  slug: so-you-need-to-give-bad-news-to-users-ryan-macklin-he-they
+- time: '13:00'
+  title: Lunch (60 mins)
+- time: '14:00'
+  title: Lightning talks (30 mins)
+- time: '14:35'
+  slug: when-documenting-is-designing-how-to-assist-api-design-as-a-technical-writer-fabrizio-ferri-benedetti
+- time: '15:20'
+  slug: adventures-in-setting-up-a-knowledge-system-for-a-research-group-victor-sluiter
+- time: '16:00'
+  title: Break (10 mins)
```

Comment:
I see what you're doing <3 I'm still not a fan of cutting talk slots shorter, and my dyslexic eyes would be really confused by all the odd time stamps... what do we think about keeping the time slots 45min and taking 15min breaks? yes it'll push the schedule forward by a little bit but I feel like it might be easier to navigate. We can also cut lunch to 45 min if we want to catch up, since most people won't go out to eat...

Reply:
I'm against cutting lunch, I think giving people the "ok" to step away from the monitor is valuable.